setup.py

```diff
@@ -15,7 +15,7 @@
     'dask[complete] >=0.18.0',
     'toolz >=0.7.4',  # ? for some dask issue (dasks does only >=0.7.3)
     'datashape >=0.5.1',
-    'numba >=0.37.0,<0.49',
+    'numba >=0.37.0',
```

Comment:
I think it should be this:

```suggestion
    'numba <0.49.0,>=0.51.0',
```

Reply:
It seems to work for version upto `numba 0.49.1`, how about 'numba <=0.49.1,>=0.51.0'

Reply:
0.49 had a major performance regression affecting datashader and was excluded for that reason.

Reply:
got it. I have pushed the fix :)